.github/workflows/continuous_production.yml

```diff
@@ -0,0 +1,47 @@
+name: Continuous production
+on:
+  - push
+  # - pull_request # already covered by push
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+          if [ -f Gopkg.toml ]; then
+              curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+              dep ensure
+          fi
```

Comment:
As of Go1.13, `go dep` is deprecated. The `ensure` command is run by `go get` before downloading a dependency.
You can replace this by `go mod tidy`, which shouldn't update `go.mod` and `go.sum`, because we will already have done that

Reply:
I'll change this then. Don't hesitate to use the "suggest code change" feature from github ;-)